### DISCUSSION TITLE
'Importing' a project

### DISCUSSION THREAD

**Discussion by humdingerb** [Genio-The-Haiku-IDE/Genio#73] [Q&A] [answered]
I'm currently using Terminal and Koder with a makefile for my projects. How do I 'convert' it to a Genio project?

**Comment by jackburton79** [comment, marked as answer]
Hi,
To open an existing "project", you use "Project" -> "Open Project" and select the directory where the code resides.

I guess this deserves a section in the readme, although in the future we could try to merge this with the regular "File" -> "Open" menu.

**Comment by humdingerb** [comment]
Damn! I could've sworn I've tried that before... :)
Thanks!

**Comment by nx6-archived** [comment]
You also need to configure the project with Project->Settings... and fill the commands and target path relevant to you.

For each flavor build or release you can set the build and clean commands and the path of the target executable.
You can add other parameters and set if the target should be opened with the Terminal.
The last one is actually very interesting. If the flag is set, the target is opened in a virtual terminal session and its stdout and sterr are captured by Genio and rerouted to the Console I/O panel.

**Comment by nx6-archived** [comment]
There is no concept of "Importing a project" in Genio, you just simply open its folder and configure it.

**Comment by humdingerb** [comment]
Nice!
It'd be cool if it detected a bog standard makefile_engine makefile and pre-fill the binary's location from Makefile's "NAME" variable and the clean and debug commands. :)
